docs/cli/graphql-transformer/directives.md

```diff
@@ -940,7 +940,7 @@ The @auth directive allows the override of the default provider for a given auth
 
 ```graphql
 # private authorization with provider override
-type Post @model @auth(rules: [{ allow: private, provider: oidc }]) {
+type Post @model @auth(rules: [{ allow: private, provider: iam }]) {
```

Comment:
Is this correct? The section is for "Authorization using an OIDC provider"

Reply:
oidc is not supported with private auth: 
This is the error message by the CLI:  
`InvalidDirectiveError: @auth directive with 'private' strategy only supports 'userPools' (default) and 'iam' providers, but found 'oidc' assigned.`

Reply:
If that's the case then you need to change more than just this code sample, the text says it's for OIDC section

Reply:
What would you suggest? 
We can remove the part that does not use the OIDC provider.